indigo_app/templates/indigo_api/_task_list_all.html

```diff
@@ -0,0 +1,55 @@
+{% load humanize indigo_app %}
+
+{% if tasks %}
+<table class="table table-striped table-bordered">
+  <thead>
+    <tr>
+      <th scope="col">Task</th>
+      <th scope="col">Place</th>
```

Comment:
Let's make the place column optional. That way, the task listing in the place view can opt-out of it.

Reply:
@longhotsummer I'm not sure I completely understand what you are asking here?

Reply:
Like we do with `selectable`, only render the Place column if the calling template tells it to do so. This will mean that in the Place Task view, we can skip the Place column (because it'll all be the same place).

Reply:
Okay, thanks. I'll update accordingly.